app/components/object-detail-widget/template.hbs

```diff
@@ -41,7 +41,12 @@
         <div>
             <h3>Identifiers</h3>
             <ul>
-                {{#each objectData._source.identifiers as |identifier|}}
+                {{#each identifiers.links as |identifier|}}
```

Comment:
Also are we meant to be displaying links that aren't clickable? I remember a convo about how it might be confusing for users